docs/src/modules/concepts/pages/concepts.adoc

```diff
@@ -114,13 +114,43 @@ Timers:: Time-based triggers
 
 == Component interoperability
 
-Akka components can communicate seamlessly, regardless of physical location. Location transparency ensures efficient message routing and runtime efficiency, even across network boundaries.
+Akka components are able to interoperate with one another regardless of where the component may be executing. While you create systems that define the relationships between each of the components in code, at runtime instances of each component can run on different nodes. The Akka runtime provides location transparency, automating the routing of a call from one component to another even if they reside in different locations on different networks.
 
-There are two main modes of inter-component communication in Akka:
+There are two ways to achieve interoperability between Akka components and the outside world.
 
-1. **ComponentClient**: Asynchronous, non-blocking invocations. Components return immediately after dispatching a message, regardless of destination.
 
-2. **Events**: Components emit events that others may subscribe to. This pattern enables decoupled, brokerless messaging similar to pub/sub systems. Akka components may also subscribe to events from external sources (e.g., APIs, brokers).
+[cols="1,3", options="header"]
+|===
+|Client Type |Description
+
+|xref:java:component-and-service-calls.adoc[ComponentClient]
+|One component can directly invoke another component. Akka treats these direct invocations as non-blocking, asynchronous messages, meaning that the invoking component immediately returns after making the invocation and can invoke any component regardless of its location.
+
+For example, a Workflow component handling a long-running business transaction can invoke an Agent component to perform a task, and the Workflow will continue processing without waiting for the Agent to complete its work.
```

Comment:
Maybe better to say that it can continue processing. 

By default, workflows steps are executed sequentially.